app-models/comments-models.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection");
+
+exports.removeCommentByCommentId = (comment_id) => {
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1`, [comment_id])
+    .then((result) => {
+      if (result.rowCount === 0){
+        return Promise.reject({
+          status: 404,
+          msg: "not found"
```

Comment:
Same as above - we can afford to be specific when we're sending custom errors back